lms/djangoapps/mobile_api/video_outlines/views.py

```diff
@@ -118,7 +119,25 @@ def get(self, request, course, *args, **kwargs):
             video_descriptor = modulestore().get_item(usage_key)
             transcripts = video_descriptor.get_transcripts_info()
             content, filename, mimetype = video_descriptor.get_transcript(transcripts, lang=lang)
-        except (NotFoundError, ValueError, KeyError):
+        except NotFoundError:
+            # Fallback mechanism for edx-val transcripts
+            transcript = get_video_transcript_content(
```

Comment:
Should this method call also be replaced with `get_val_transcript`?

Reply:
no response is different here.